src/ucp/proto/proto_am.inl

```diff
@@ -602,4 +617,20 @@ ucp_am_bcopy_handle_status_from_pending(uct_pending_req_t *self, int multi,
     return UCS_OK;
 }
 
+static UCS_F_ALWAYS_INLINE ucs_status_t
+ucp_tag_send_am_short_iov(ucp_ep_h ep, const void *buffer, size_t length,
```

Comment:
ucp_tag_send_eager_compat_short_iov
and move to src/ucp/tag 
since this func uses ucp_eager_compat_hdr_t explicitly